queue_job_subscribe/i18n/ca.po

```diff
@@ -29,11 +30,13 @@ msgid ""
 "If this flag is checked and the user is Connector Manager, he will receive "
 "job notifications."
 msgstr ""
+"Si aquesta marca està marcada i l'usuari és Connector Manager, rebrà "
```

Comment:
```suggestion
"Si aquesta casella de selecció està marcada i l'usuari és Connector Manager, rebrà "
```

In Catalan, _checkbox_ should be translated as _[casella de selecció](https://ca.wikipedia.org/wiki/Casella_de_selecci%C3%B3)_, according to [TERMCAT](https://www.termcat.cat/ca/cercaterm/casella%20de%20selecci%C3%B3?type=basic&thematic_area=Tecnologies%20de%20la%20informaci%C3%B3%20i%20la%20comunicaci%C3%B3&language=ca).

Reply:
That should be done in [weblate](https://translation.odoo-community.org/), though: it's not the point of the migration